mocks/consumer_service.go

```diff
@@ -0,0 +1,72 @@
+package mocks
```

Comment:
In order to run this mock, you must have a server running on port 8080. In my opinion, this does not qualify as a Mock. Mocks should be implemented as simply responding to an interface.
https://en.wikipedia.org/wiki/Mock_object

I would recommend improving it to the following structure

Create an interface for sending files to the consumer.

This interface could be organized as follows.

```
type FileStorage interface {
    Add(dealID int64, dataHash string, data []byte) string
}
```

Then create an implementation of Storage for the ConsumerService.
```
type ConsumerServiceFileStorage struct {

}

func (s *ConsumerServiceFileStorage) Add(dealID int64, dataHash string, data []byte) string {
   postData...
}
```

This minimizes code changes if you decide to add IPFS or S3 in the future.
Of course, integrating IPFS or S3 will require more consideration, but that's not the point here.
What I'm saying is that if you want to use Mock, you'll want to interface it.

Reply:
And it looks like the mock could be created like this.

```
type MockFileStorage struct {}

func (s *MockFileStorage) Add(dealID int64, dataHash string, data []byte) string {
   return "expectedId"
}
```

Reply:
@gyuguen I've made some changes to the `consumer service` and mock service to reflect your comments, if you'd like to check it out, I'd appreciate it.